examples/dotnet/SeleniumDocs/Bidirectional/BidiApiTest.cs

```diff
@@ -72,5 +93,23 @@ public async Task InterceptNetworkRequest()
 
             StringAssert.Contains(driver.PageSource, "delicious cheese");
         }
+
+        private string IsDisplayedScript()
```

Comment:
I guess no private methods should be in examples. Any example should be self-readable. It's better to provide simple js code to be executed directly in example, rather than hide it from user as private.

Reply:
This isn't going to be displayed in the docs page. Before his and private methods that are obvious should be fine. We could do a trivial example that isn't as interesting, but this one is especially useful in the real world, so I prefer it.

Reply:
Ok, I removed this after all to make it more simple